src/main/java/cloud/fogbow/ras/core/plugins/interoperability/azure/attachment/AzureAttachmentPlugin.java

```diff
@@ -0,0 +1,171 @@
+        return doRequestInstance(azure, virtualMachineId, diskId);
+    }
+
+    @Override
+    public AttachmentInstance getInstance(AttachmentOrder attachmentOrder, AzureUser azureUser) throws FogbowException {
+        LOGGER.info(String.format(Messages.Info.GETTING_INSTANCE_S, attachmentOrder.getInstanceId()));
+        Azure azure = AzureClientCacheManager.getAzure(azureUser);
+        String subscriptionId = azureUser.getSubscriptionId();
+        String resourceId = buildResourceId(subscriptionId, attachmentOrder.getInstanceId());
+        
+        return doGetInstance(azure, resourceId);
+    }
+    
+    @Override
+    public void deleteInstance(AttachmentOrder attachmentOrder, AzureUser azureUser) throws FogbowException {
+        LOGGER.info(String.format(Messages.Info.DELETING_INSTANCE_S, attachmentOrder.getInstanceId()));
+        Azure azure = AzureClientCacheManager.getAzure(azureUser);
+        String subscriptionId = azureUser.getSubscriptionId();
+        String resourceId = buildResourceId(subscriptionId, attachmentOrder.getInstanceId());
+        String virtualMachineId = buildResourceId(subscriptionId, attachmentOrder.getComputeId());
+        
+        doDeleteInstance(azure, virtualMachineId, resourceId);
+    }
+    
+    @VisibleForTesting
+    void doDeleteInstance(Azure azure, String virtualMachineId, String resourceId) throws FogbowException {
+        VirtualMachine virtualMachine = doGetVirtualMachineSDK(azure, virtualMachineId);
+        VirtualMachineDataDisk virtualMachineDataDisk = findVirtualMachineDataDisk(virtualMachine, resourceId);
+        int lun = virtualMachineDataDisk.lun();
+        
+        Observable<VirtualMachine> observable = AzureAttachmentSDK.detachDisk(virtualMachine, lun);
+        this.operation.subscribeDetachDiskFrom(observable);
+    }
+
+    @VisibleForTesting
+    VirtualMachineDataDisk findVirtualMachineDataDisk(VirtualMachine virtualMachine, String resourceId)
+            throws FogbowException {
+        
+        return virtualMachine.dataDisks().values().stream()
+                .filter(dataDisk -> resourceId.equals(dataDisk.id()))
+                .findFirst()
+                .orElseThrow(() -> new InstanceNotFoundException(Messages.Exception.INSTANCE_NOT_FOUND));
+    }
+    
+    @VisibleForTesting
+    AttachmentInstance doGetInstance(Azure azure, String resourceId) throws FogbowException {
+        Disk disk = doGetDiskSDK(azure, resourceId);
+        return buildAttachmentInstance(disk);
+    }
+    
+    @VisibleForTesting
+    AttachmentInstance buildAttachmentInstance(Disk disk) {
+        DiskInner diskInner = disk.inner();
+        String id = diskInner.id();
+        String cloudState = disk.isAttachedToVirtualMachine() 
+                ? AzureStateMapper.ATTACHED_STATE
+                        : AzureStateMapper.UNATTACHED_STATE;
+        
+        String computeId = disk.virtualMachineId();
+        String volumeId = disk.id();
+        String device = null;
```

Comment:
Use the constant dedicated to this kind of value(without information). 
Look at this link: https://github.com/fogbow/resource-allocation-service/blob/azure-attachment-plugin/src/main/java/cloud/fogbow/ras/core/plugins/interoperability/azure/util/AzureGeneralUtil.java#L10

Reply:
Ok, I'll fix it.